docs/application/native/guides/multimedia/media-controller.md

```diff
@@ -250,6 +669,23 @@ The following table lists all the server metadata attributes the client can rece
 | `MC_META_MEDIA_TRACK_NUM`   | Track number of the latest content in the media controller server |
 | `MC_META_MEDIA_PICTURE`     | Album art of the latest content in the media controller server |
 
+<a name="playlist_updatemode"></a>
```

Comment:
As mentioned earlier, please avoid using HTML code. Use only MD specific code.

Reply:
The 'playlist_updatemode' tag is not used. So I removed it.